lib/voice_manager.py

```diff
@@ -16,11 +16,17 @@ def delete(self, guild_id):
             pass
 
     async def close(self, guild_id, text="読み上げを終了します。"):
-        await self.servers[guild_id].close(text)
+        try:
+            await self.servers[guild_id].close(text)
+        except Exception:
+            pass
         self.delete(guild_id)
 
     async def all_close(self, text="読み上げを終了します。"):
         for key in list(self.servers):
             server = self.servers[key]
```

Comment:
```suggestion
        for key, server in self.servers.items():
```
今回の変更とは全く関係無いんですけど、辞書型をリスト型にキャストする意図は何かありますか？

Reply:
```py
>>> d = {1: "hello", 2: "world"}
>>> for key, value in d.items():
...     del d[key]
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: dictionary changed size during iteration
```

このようにdictをそのままforで回している間にdelなどをするとエラーになってしまうのでそれを防いでいます。
確か`server.close(text)`の中でdelしているからだと思います

Reply:
あ、`server.close(text)`ではなく`self.delete(key)`があるからかもしれません:pray:

Reply:
あぁ、見落としてました
`for key, server in self.servers.copy().items()`になるのか
微妙ですね…

Reply:
そうですね〜やはりlistにしたほうが綺麗だと思います